MatrixSDK/MXSession.h

```diff
@@ -503,6 +503,16 @@ FOUNDATION_EXPORT NSString *const kMXSessionNoRoomTag;
  */
 @property (nonatomic, readonly, getter=isPauseable) BOOL pauseable;
 
+/**
+ Whether the user is part of a room with the membership state of `join` or
+ they are in the process of joining.
+
+ @param roomIdOrAlias The ID or alias of the room to check.
+
+ @return YES if they are.
+ */
+- (BOOL)isJoinedOnRoom:(NSString *)roomIdOrAlias;
```

Comment:
Doesn't have an implementation.

Reply:
Ah.... This is exposing an existing private function 🙂